.changeset/short-bananas-share.md

```diff
@@ -0,0 +1,6 @@
+---
+'vercel': minor
+'@vercel/error-utils': patch
```

Comment:
I don't see any changes to @vercel/error-utils. I don't think we need to bump version here.

Reply:
Yeah, not sure why this is there. Weird.